MCPTT/UI/Contact/Contact List/ContactListViewController.swift

```diff
@@ -56,124 +70,80 @@ final class ContactListViewController : UIViewController, UICollectionViewDelega
         
         viewModel.reloadCollectionViewClosure = { [weak self] () in
             DispatchQueue.main.async {
-                self?.collectionView.reloadData()
+                self?.tableView.reloadData()
             }
         }
     }
-} //class ending
-
-extension ContactListViewController: UICollectionViewDelegate, UICollectionViewDataSource {
-    
-    // MARK: UICollectionViewDataSource
-    
-    public func numberOfSections(in collectionView: UICollectionView) -> Int {
-        return 3
-    }
-    
-    public func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
-        switch section {
-        case 0:
-            return CGSize.init(width: screenSize.width, height: 30)
-        case 1:
-            return CGSize.init(width: screenSize.width, height: 30)
-        default:
-            return CGSize.init(width: screenSize.width, height: 30)
-        }
-    }
     
-    public func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-        
-        let view = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionElementKindSectionHeader, withReuseIdentifier: "Header", for: indexPath) as? SectionHeaderCell
-        switch (indexPath.section) {
-        case 0:
-            view?.headerNameLabel.text = "Me"
-            return view ?? UICollectionReusableView()
-        case 1:
-            view?.headerNameLabel.text = "Groups"
-            return view ?? UICollectionReusableView()
-        default:
-            view?.headerNameLabel.text = "A  "
-            return view ?? UICollectionReusableView()
-        }
+    func numberOfSections(in tableView: UITableView) -> Int {
+         return contactSectionTitlesUpdated.count
     }
     
-    public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         switch section {
         case 0:
             return 1
         case 1:
             return groups.count
         default:
-            return viewModel.numberOfCells
+            let contactKey = contactSectionTitlesUpdated[section]
+            if let contactValue = contactNameDictionary[contactKey] {
+                return contactValue.count
```

Comment:
change the variable name to be self explanatory

Reply:
done the changes